conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+BinderHubs can showcase the reproducibility of software as well as being a valuable teaching or demonstration tool.
+
+In this workshop, attendees will learn how to deploy a BinderHub on Microsoft Azure and connect with others who have or are interested in doing so.
+Attendees should be familiar with Azure Cloud computing and the command line.
+
+### Audience
+
+_Would your target audience be required to have any prerequisite skills/background knowledge e.g. knowledge of a particular language?_
+
+Attendees would preferably be familiar with Microsoft Azure and the command line.
+
+### Outcomes
+
+_How will your attendees benefit from your session? What do you expect them to gain/learn?_
+
+_(600 characters max, current count: 524 [please update if you edit])_
+
+Attendees to this workshop will learn how to deploy a BinderHub on Microsoft Azure Cloud platform.
+They will learn the multiple applications of a BinderHub as a teaching or demonstration tool; the technology powering a BinderHub; and why reproducibility is important in research.
+Attendees will be provided with enough knowledge to make an informed decision on whether building their own BinderHub is advantageous to their goals and connect with other RSEs/researchers who have build a BinderHub or are interested in doing so.
+
+### Accessibility
+
+_How will you ensure that your content will be accessible to the diverse audience of the conference?_
+
+### Promotion
+
+_How will you attract a broad and diverse audience for your session?_
+
+I hope to attract attendees by being a visible, active and welcoming member of the RSE community.
+I especially hope that by giving this workshop, I encourage more women, members of the LBGTQ+ and non-binary communities to engage with Cloud computing projects and open source communities.
+
+### Mentorship
+
+_Optional section._
+
+_Would you like a mentor? If so then what would you like the mentor to do?_
+
+No?
+
+### Resources and accessibility needs
+
+_Optional section._
+
+_What technological requirements do you have for your session? Do you have any accessibility needs that we should consider when planning when and where to include your contribution?_
+
+Attendees will require:
+* a "Free Trial" subscription to Azure;
+* a valid DockerHub account.
+
+If they are able to install software before the event, that would be preferred:
+* Azure CLI - https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest
+* Kubernetes CLI (`kubectl`) - https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos
+* Helm (Kubernetes package manager) - https://helm.sh/docs/using_helm/#installing-helm
+
+### Will you be applying for a travel bursary?
+
+No.
+
+### Licence
+
+_All materials uploaded and generated for your session will be published on the conference app and website will be shared under a CCBY license._
+_This is a requirement for all presenters._
+_Do you agree to this?_
+
+Yes.
```

Comment:
Can you add the other questions here? I know you don't need my review on them but its quite an interesting form to capture 😃

Reply:
Can do. I didn't include them as I wasn't going to answer them! I just captured the mandatory fields and any optional ones I thought were relevant.

Reply:
Sorry, all the questions I've left out are mandatory, just check boxes 🤦‍♀ Will add them!

Reply:
Done in [`b4a09fb`](https://github.com/alan-turing-institute/the-turing-way/pull/425/commits/b4a09fbae81de0535408f75128dabecdc308f3f1)